### PR TITLE
Styling Updates

### PR DESCRIPTION
**Pull request by lydia-zheng**
- Made Map larger
- Cleaned up ShareCalendar button styling
- Added basic styling to DetailModal (**Let me know if you guys have preferences for colors**)
-  Added description labels in EventDetailsModal
- EventDetailModal currently responsive(buttons fit inside modal)
- Unified Title style format for ModalNewEvent

### CODE REVIEW

File: parkhangs-webapp/src/features/modal/ModalDetail.css [hunk 1]
```diff
@@ -15,9 +15,9 @@
 .modal-container {
     z-index: 101;
     background-color: white;
-    width: 70%;
+    width: 80%;
     box-shadow: 0 2px 5px 0 rgba(0,0,0,0.16), 0 2px 10px 0 rgba(0,0,0,0.12);
-    border-radius: 0.1875rem;
+    /* border-radius: 0.1875rem; */
```
Comment (cmaija) [inline parkhangs-webapp/src/features/modal/ModalDetail.css:20]: please just delete code if you don't want it there

File: parkhangs-webapp/src/features/modal/ModalDetail.css [hunk 2]
```diff
@@ -77,10 +77,39 @@
...
 
 .EventsList-populated{
     max-height: 31rem;
     overflow: scroll
 }
+
+@media only screen and (max-width:600px){
+    .modal-container {
+        padding-top: 0;
+        background-color: #E3F1DF;
```
Comment (cmaija) [inline parkhangs-webapp/src/features/modal/ModalDetail.css:93]: is there a reason you're changing the background color when the screen size changes

Comment (lydia-zheng) [reply parkhangs-webapp/src/features/modal/ModalDetail.css:93]: Oh I was testing using different colors for the two queries initially, I can move it out of the queries if that bothers you

File: parkhangs-webapp/src/components/ShareCalendar.js [hunk 3]
```diff
@@ -13,11 +14,19 @@ class ShareCalendar extends React.Component {
   }
  
   render() {
-    let icon = { 'calendar-plus-o': 'left' };
+    let icon= { 'calendar-plus-o': 'left' };
+    let items= [ 
+      {google: 'Google'},
```
Comment (gordonzhang17) [inline parkhangs-webapp/src/components/ShareCalendar.js:19]: we dont want the other ones (like Outlook, Yahoo?)

File: parkhangs-webapp/src/features/modal/ModalEventDetail.js [hunk 4]
```diff
@@ -104,13 +104,13 @@ class ModalEventDetail extends React.Component {
         return (
             <div className="ModalEventDetail-description">
                 <div className="ModalEventDetail-description-section">
-                    <span>{event.favoritesCount || 0} people have favorited this event</span>
+                    <span><b>Event Favorites:</b> {event.favoritesCount || 0} people have favorited this event</span>
```
Comment (gordonzhang17) [inline parkhangs-webapp/src/features/modal/ModalEventDetail.js:107]: *liked/favourited

File: parkhangs-webapp/src/features/modal/ModalEventDetail.js [hunk 5]
```diff
@@ -255,15 +255,15 @@ class ModalEventDetail extends React.Component {
                                 this.props.user != null &&
                                 <button
                                     onClick={() => this.favouriteEvent(event._id)}
-                                    className="ModalEventDetail-actionButton">
+                                    className="ModalEventDetail-actionButton faveEvent">
```
Comment (gordonzhang17) [inline parkhangs-webapp/src/features/modal/ModalEventDetail.js:258]: can we not use shorthand like this?

File: parkhangs-webapp/src/components/ShareCalendar.js (hunk 3 above)
Comment (lydia-zheng) [reply parkhangs-webapp/src/components/ShareCalendar.js:19]: I can add them back in if we still want them, but I thought the drop down would be cleaner with fewer options. Thought last time someone implied we don't need all the options.

File: parkhangs-webapp/src/features/modal/ModalEventDetail.js (hunk 5 above)
Comment (lydia-zheng) [reply parkhangs-webapp/src/features/modal/ModalEventDetail.js:258]: fixed

File: parkhangs-webapp/src/components/ShareCalendar.js (hunk 3 above)
Comment (gordonzhang17) [reply parkhangs-webapp/src/components/ShareCalendar.js:19]: haha I think that was me. Em up to you, we can always add them back on later but it does't matter for now.

**Pull request merged**